js/views/components/moderators/Moderators.js

```diff
@@ -93,9 +94,9 @@ export default class extends baseVw {
 
     super(opts);
     this.options = opts;
-    this.excludeIDs = opts.excludeIDs;
+    this.excludeIDs = [...opts.excludeIDs, app.profile.id];
```

Comment:
It seemed a lot smarter just include the user's own profile here, instead of adding it later.

We could add a parameter to not exclude it in the future if that use case ever came up.